snowplow/iglu-client-embedded/schemas/com.metabase/structured_action/jsonschema/1-0-0

```diff
@@ -0,0 +1,63 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Generic event to track interactions and actions that happen within Metabase. This should be used as a default schema for tracking events. If more specific context is needed, we can use custom context entities (https://docs.snowplow.io/docs/understanding-your-pipeline/entities/).",
+    "self": {
+      "vendor": "com.metabase",
+      "name": "structured_action",
```

Comment:
maybe just `action`?

Reply:
Yes I wanted to do so as well but we already have an `action` schema unfortunately. But that legacy one is more a "category" of actions combined into a single schema. A potential alternative approach would be to create a new major version of the old `action` schema but I am not sure whether this would be more confusing.

Reply:
ok, makes sense to keep this as is.